pisp_app/lib/Screens/transaction_details.dart

```diff
@@ -0,0 +1,294 @@
+import 'package:flutter/material.dart';
+import 'package:logger/logger.dart';
+import 'package:pispapp/MockData/account.dart';
+import 'package:pispapp/Screens/dashboard.dart';
+import 'package:pispapp/Screens/transaction_success.dart';
+
+import 'package:pispapp/log_printer.dart';
+import 'package:local_auth/local_auth.dart';
+import 'package:flutter/services.dart';
+
+class TransactionDetails extends StatefulWidget {
+  const TransactionDetails(this.payerAccount, this.payeeAccount, this.amount);
+
+  final Account payerAccount, payeeAccount;
+  final String amount;
+
+  @override
+  _TransactionDetailsState createState() => _TransactionDetailsState();
+}
+
+class _TransactionDetailsState extends State<TransactionDetails> {
+  bool _isLoading = true;
+  final LocalAuthentication _localAuthentication = LocalAuthentication();
+
+  String charges;
+
+  // Simulates api call
+  // TODO(MahidharBandaru): To be moved out
+  Future<String> sleep1() {
+    return Future<String>.delayed(const Duration(seconds: 1), () => '10');
+  }
+
+  Future<void> getTransactionCharges() async {
+    charges = await sleep1();
+    setState(() {
+      _isLoading = false;
+    });
+  }
+
+  Future<bool> _isBiometricAvailable() async {
+    bool isAvailable = false;
+    try {
+      isAvailable = await _localAuthentication.canCheckBiometrics;
+    } on PlatformException catch (e) {
+      print(e);
+    }
+
+    if (!mounted) {
+      return isAvailable;
+    }
+
+    isAvailable
+        ? print('Biometric is available!')
+        : print('Biometric is unavailable.');
+
+    return isAvailable;
+  }
+
+  Future<void> _getListOfBiometricTypes() async {
+    List<BiometricType> listOfBiometrics;
+    try {
+      listOfBiometrics = await _localAuthentication.getAvailableBiometrics();
+    } on PlatformException catch (e) {
+      print(e);
+    }
+
+    if (!mounted) {
+      return;
+    }
+
+    print(listOfBiometrics);
+  }
+
+  Future<void> _authenticateUser() async {
```

Comment:
Can you consider to add a support for passcode authentication? I only tested the app using a simulator and it doesn't seem to support fingerprint... hence, I couldn't even try to make a transaction :')

Reply:
I'll add this in another PR ?

Reply:
Okay, thank you!